clang/tools/3c/README.md

```diff
@@ -0,0 +1 @@
+TODO
```

Comment:
What's supposed to be in this one, compared the info in `clang/docs` ?

Reply:
How to use the `3c` tool.  This is now being reviewed, so I'm going to resolve this conversation.